source/sass/views/initiatives.scss

```diff
@@ -1,7 +0,0 @@
-.bg-white {
-  background: $white;
-}
-
-.bottom-line {
```

Comment:
Ruleset has been relocated to `source/sass/views/participate.scss` as participate page is the only page that uses this class.